src/mrsimulator/method/__init__.py

```diff
@@ -383,6 +478,71 @@ def get_transition_pathways(self, spin_system) -> List[TransitionPathway]:
             for item in segments
         ]
 
+    def events_to_dataframe(self, properties=None) -> pd.DataFrame:
+        """Returns dataframe of requested Event properties
+
+        Args:
+            List properties: properties to include in df columns. Include all if empty
+
+        Returns:
+            pd.DataFrame df: properties as columns and event number as row
+
+        Example:
+            TODO add example code
+        """
+        # TODO: Add catch for empty 'spectral_dimensions' and 'events'
+        spec_dims = self.spectral_dimensions
+        gsp = self.get_symmetry_pathways
+
+        if properties is None:
+            properties = [
+                "magnetic_flux_density",
+                "rotor_frequency",
+                "rotor_angle",
+                "p",
+                "d",
```

Comment:
@mgiammar Can you also include ``freq_contrib`` in this data frame object? ``freq_contrib`` doesn't need to be plotted. 

Also, since this function is a tabulated summary of the method object, why not name it ``summary``. What are your thoughts?

Reply:
Adding `freq_contrib` is easy.

As for method name do you think `summary_as_df` or `summary_as_dataframe` would be better? Or would that confuse users into thinking there is another summary method which does not return a df?

Reply:
Let's go with ``summary``. ``summary_as_df`` or ``summary_as_dataframe `` are explicit, but as long as you specify the return type of the function in the docs, it doesn't need to be in the function name.